frame/pablo/src/lib.rs

```diff
@@ -573,6 +634,107 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		fn default_pblo_staking_pool_config(
+			pool_id: &T::PoolId,
+			pair: &CurrencyPair<T::AssetId>,
+		) -> Result<
+			RewardPoolConfiguration<
+				T::AccountId,
+				T::AssetId,
+				T::BlockNumber,
+				RewardConfigsOf<T>,
+				DurationPresets<T>,
+			>,
+			DispatchError,
+		> {
+			let mut rewards = BTreeMap::new();
+			let max_rewards: T::Balance = T::Convert::convert(MAX_REWARDS);
+			let reward_rate = Perbill::from_percent(REWARD_PERCENTAGE);
+			rewards
+				.insert(pair.base, RewardConfig { asset_id: pair.base, max_rewards, reward_rate });
+			rewards.insert(
+				pair.quote,
+				RewardConfig { asset_id: pair.quote, max_rewards, reward_rate },
+			);
+			let reward_configs = RewardConfigsOf::<T>::try_from(rewards)
+				.map_err(|_| Error::<T>::StakingPoolConfigError)?;
+			let mut duration_presets = BTreeMap::new();
+			duration_presets.insert(ONE_HOUR, Perbill::from_percent(1));
```

Comment:
These are not really good defaults, staking for a minute or an hour is not practical. We can set something like
DAY, WEEK, MONTH, 3MONTH, YEAR etc